CHANGES.rst

```diff
@@ -23,6 +23,11 @@ Unreleased
 * :meth:`.Comment.parent` to obtain the parent of a :class:`.Comment`.
 * :meth:`.opt_in` and :meth:`.opt_out` to :class:`.Subreddit` to permit working
   with quarantined subreddits.
+* :class:`praw.models.LiveUpdate`.
+* :meth:`praw.models.LiveUpdate.thread` to get the parent live thread of the
```

Comment:
Since LiveUpdate is added in this changelog, I don't think we need to detail the added methods.